rpcs3/Emu/RSX/GL/GLTextureCache.h

```diff
@@ -495,7 +495,7 @@ namespace gl
 			const u32 valid_length = valid_range.second;
 			AUDIT( valid_length > 0 );
 
-			void *dst = get_ptr_by_offset(valid_range.first, true);
+			void *dst = get_ptr(get_section_base());
```

Comment:
I believe this is incorrect.
`get_section_base()` returns the section base (`cpu_range.start`), but we want `cpu_range.start + valid_offset`.

Should be `get_ptr(get_section_base() + valid_offset)`, I think

Reply:
Ok, fixed.